public/swagger-ui/css/print.css

```diff
@@ -0,0 +1,1367 @@
+/* Original style from softwaremaniacs.org (c) Ivan Sagalaev <Maniac@SoftwareManiacs.Org> */
```

Comment:
Yes, we added an official VuFind Gruntfile to master quite recently (in time for release 3.1), so you should have an existing foundation to add tasks to.